draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -939,30 +939,17 @@ following conditions holds:
 
 ## "Same-site" and "cross-site" Requests  {#same-site-requests}
 
-Two origins, A and B, are considered same-site if the following algorithm
-returns true:
+Two origins are same-site if they satisfy the "same site" criteria defined in
+{{HTML}}. A request is "same-site" if the following criteria are true:
 
-1.  If A and B are both the same globally unique identifier, return true.
+1.  The request is not the result of a cross-site redirect. That is,
+    the origin of every url in the request's url list is same-site with the
+    request's current url's origin.
 
-2.  If A and B are both scheme/host/port triples:
+2.  The target's URI's origin is same-site with the request's client's "site
+    for cookies" (which is an origin), or if the request has no client.
```

Comment:
Trying to understand what it means to not have a client, https://fetch.spec.whatwg.org/#concept-request-client

Should this be phrased `request client is null` instead of `request has no client`? Or does it mean something else?

Reply:
I don't think we should update this to `request client is null` because there are requests from user agents that actually don't have a notion of a client.

I was debating whether we need to tweak the rest of the definition when reading with that lens, but I think the cross-site redirect restriction (bullet point 1) should still apply to a request from a user agent that doesn't have a client.

Reply:
@chlily1 made a good point I think (offline), can we say something like ~"is null or has no client"? That sounds good to me.

Reply:
SGTM. I've added that in

Reply:
looks good, thanks